.github/workflows/test.yml

```diff
@@ -1,32 +1,32 @@
-name: Test
-
-on:
-  pull_request:
-    branches:
-      - "master"
-
-jobs:
-  test:
-    runs-on: ubuntu-latest
-    timeout-minutes: 10
-
-    steps:
-      - name: Checkout repo
-        uses: actions/checkout@v2
-
-      - name: Set Up Python 3.10
-        uses: actions/setup-python@v2
-        with:
-          python-version: "3.10"
-
-      - name: Install requirements.txt
-        run: |
-          python -m pip install --upgrade pip
-          pip install -r requirements.txt
-
-      - name: Run flake8
-        run: flake8
-
-      - name: Run tests
-        timeout-minutes: 5
-        run: python manage.py test
+name: Test
+
+on:
+  pull_request:
+    branches:
+      - "master"

```

Comment:
The workflow is currently set to trigger only on pull requests to the 'master' branch. Ensure this aligns with your branching strategy. If you have other branches that require testing, consider adding them here.